README.md

```diff
@@ -44,6 +44,21 @@ following actions on the project and its submodules:
 
 > **_NOTE:_** The minimum supported Gradle version to use this plugin is v6.9
 
+<details>
+<summary> 🚚 Migrating from v4 </summary>
+
+### Breaking Changes 🚨
+
+* The minimum version of Gradle required to use this plugin has been upgraded to version 6.9.
+* Artifactory's convention attributes have been removed:
```

Comment:
Artifactory's convention attributes have been removed:
-->
The below convention attributes have been removed: